tests/virt_autotest/libvirt_virtual_network_init.pm

```diff
@@ -71,6 +71,7 @@ sub run_test {
         script_retry("nmap $guest -PN -p ssh | grep open", delay => 30, retry => 6, timeout => 180);
         save_guest_ip($guest, name => "br123");
         virt_autotest::utils::ssh_copy_id($guest);
+        check_guest_health($guest);
```

Comment:
Quick question, as the same, would you mind adding helpful info here. I still totally no clear why add check_guest_health($guest); at libvirt virtual network test init . thanks. is it the debug way for SRIOV failure, right?

Reply:
I replied on previous comment.